tests/e2e/cli/semantic_segmentation/test_segmentation.py

```diff
@@ -317,6 +317,8 @@ class TestToolsMPASelfSLSegmentation:
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
     def test_otx_train(self, template, tmp_dir_path):
         tmp_dir_path_1 = tmp_dir_path / "segmentation/test_selfsl"
+        if not os.path.exists(os.path.join(template.model_template_path[:-14], "selfsl")):
```

Comment:
We agreed to use Pathlib in OTX. Also, `path.parent` may solve the slitting issue without indexing lists